src/context/reducers/notificationReducer.tsx

```diff
@@ -10,40 +10,44 @@ export enum NotificationItemVariant {
 }
 
 export interface NotificationItem {
-  variant: 'success' | 'danger' | 'warning' | 'info' | 'default'
-  title: string;
-  message: string;
-  timestamp: Moment;
+  variant: 'success' | 'danger' | 'warning' | 'info' | 'default', //here need? how does it work? what's it doing? remove?
+  title: string,
+  message: string,
+  timestamp: Moment
```

Comment:
Even though using both commas and semicolons are valid for interfaces, the preferred separator in TypeScript for interfaces is semicolons. 

Please, replace it with semicolons since once linting is enabled with best practices of typescript there will be code inconsistency warnings